content/03.use-cases-a.md

```diff
@@ -47,3 +47,10 @@ There are (as always) no "points" awarded for using ALL GitHub materials ALL the
 
 Matthew D. Beckman, Mine Çetinkaya-Rundel, Nicholas J. Horton, Colin W. Rundel, Adam J. Sullivan & Maria Tackett (2021) Implementing Version Control With Git and GitHub as a Learning Objective in Statistics and Data Science Courses, Journal of Statistics and Data Science Education, 29:sup1, S132-S144, DOI: 10.1080/10691898.2020.1848485
 
+
+Github is a valuable tool/platform to support both learning and teaching as a submission platform (to host their work and submit their course assignments and
+projects). Therefore Github can be used as a way to host course content, through basic GitHub, the organizations feature, and GitHub classroom.Therefore teachers can utilize GitHub as typical learning management systems to improve assignment submission, grading and feedback systems [@doi:10.2190/1002-1T50-27G2-H3V7].
+It is also important to consider that GitHub provides a graph view to visualize and summarize students project activities and reports[@doi:10.1145/2675133.2675284]. Therefore, by visualizing a summary of classroom and project activities; a graph view allowed instructors/trainers and teachers to easy obtain well-detailed view of trainee and student class activity which his provided by the commit history. Moreover, version control as a part of Github can help students for their future work. students can benefit from Github in which Students with experience using version control are more prepared and qualified to work in industry[@doi:10.1145/2445196.2445486].
+
+Saeed: I hope it is updated online too :)
```

Comment:
Use HTML comments to include comments but prevent them from showing up in the rendered manuscript:
```suggestion
<!--Saeed: I hope it is updated online too :)-->
```

Reply:
> Use HTML comments to include comments but prevent them from showing up in the rendered manuscript:

Thank you @Aariq  comment is now deleted accordingly.